src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.java

```diff
@@ -0,0 +1,53 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NeedBraces"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.needbraces;
+
+// xdoc section -- start
+class Example1 {
+  String obj = new String();
+  public boolean Example1() {
+    int num = 12;
+    if (obj.equals(num)) return true;
+    // violation above, ''if' construct must use '{}'s.'
+    // single-line statements not allowed without braces.
+    int count = 0;
+    if (true) {
+      count = 2;
+    } else
+        // violation above, ''else' construct must use '{}'s.'
+        // single-line statements not allowed without braces.
+        return false;
+    for (int i = 0; i < 5; i++) {
+      ++count;
+    }
+    do  // single-line statements not allowed without braces.
+        // violation above, ''do' construct must use '{}'s.'
+        ++count;
+    while (false);
+
+    for (int j = 0; j < 10; j++); // empty loop body not allowed.
```

Comment:
Please remove extra empty lines, line you have in line 33.
Examples should be the same in size and code, and only violation comments changing their location from example to example.

Reply:
To keep the same file size, In some examples I have kept some extra lines to maintain the file size of 46 lines. Share any more enhancements if required !

Reply:
yes